common/lib/xmodule/xmodule/tests/test_graders.py

```diff
@@ -282,9 +261,9 @@ def test_grade_with_string_min_count(self):
         ])
 
         graded = weighted_grader.grade(self.test_gradesheet)
-        self.assertAlmostEqual(graded['percent'], 0.50812499999999994)
-        self.assertEqual(len(graded['section_breakdown']), (12 + 1) + (7 + 1) + 1)
-        self.assertEqual(len(graded['grade_breakdown']), 3)
+        assert round(graded['percent'] - 0.5081249999999999, 7) >= 0
```

Comment:
It is skipping the last digit from the original value, but it shouldn't be a problem. The same change is also present in the next assertions.